app/controllers/EntryDetailsController.scala

```diff
@@ -47,6 +47,10 @@ class EntryDetailsController @Inject()(
   def onPageLoad(mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) {
     implicit request =>
 
+      if(request.userAnswers.get(CustomsRegulationTypePage) == Some(CustomsRegulationType.UnionsCustomsCodeRegulation)) {
```

Comment:
Sorry, just spotted this and giving it some thought, I'm not sure if this is the right place to set this answer. Could we not set the user answers on submit of CustomsRegluationTypePage? Also, I think the ArticleType should be set to Schedule if the user selects UKCustomsCodeRegulation.

Reply:
This review comment is incorporated.